test/e2e/etcd_operator_unannotated_test.go

```diff
@@ -0,0 +1,78 @@
+package e2e
+
+import (
+	"context"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+	"time"
+)
+
+func ServiceBindingRequestSetup(t *testing.T, steps []Step) {
```

Comment:
Is the same method we have in the other end-to-end test file? Could we share the same code among those?

Reply:
Outdated